meta-intel-edison-distro/conf/distro/poky-edison.conf

```diff
@@ -6,17 +6,17 @@ PREFERRED_VERSION_linux-yocto = "4.16.0%"
 PREFERRED_PROVIDER_virtual/bootloader ?=  "u-boot"
 PREFERRED_VERSION_u-boot ?= "2018.05"
 PREFERRED_VERSION_u-boot-fw-utils ?= "2018.05"
-PREFERRED_VERSION_pulseaudio ?= "10.0%"
-PREFERRED_VERSION_openssl ?= "1.0.2%"
-PREFERRED_VERSION_systemd ?= "234%"
-PREFERRED_VERSION_nodejs = "6.%"
-PREFERRED_VERSION_nodejs-native = "6.%"
+PREFERRED_VERSION_pulseaudio ?= "11.1%"
```

Comment:
We could remove preferred versions here. Maybe not for node-js.

Reply:
Well, I'd actually leave this here and remove e.g. the kernel-ones from the bsp layer's layer.conf (which IMHO is non-intuitive location). Having all preferred versions in one place makes sense to me. Or you mean that after unused layers cleanup these are not necessary? I can double-check that when preparing my PR.

Reply:
Eh, I meant do we have a preferred version of f.i. systemd? Or just the latest available?

Reply:
Oh, that one I think is yes - just whatever latest we've got, I'm not aware of any specific version dependencies.